### PR TITLE
chore:update-notifications-panel-version-to-0.0.4

### PR DESCRIPTION
**Pull request by Enzo707**
This PR migrates notifications-panel version from 0.0.3 to 0.0.4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "url": "git+https://github.com/DynamoDS/NotificationCenter.git"
   },
   "publishConfig": {
-    "access": "public"
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
```
Comment (QilongTang) [inline package.json:14]: What is the reason to add this line?

Comment (Enzo707) [reply package.json]: I added this to publish the npm package manually before I realized we got a pipeline. I'll remove it

**Comment by QilongTang** [comment]
@Enzo707 @avidit any insights on why the PR check would fail?

**Comment by reddyashish** [comment]
I don't think there should there be any changes to package-lock json in the PR. Just the version update in the package.json. I think that's why it is failing

**Comment by avidit** [comment]
> @Enzo707 @avidit any insights on why the PR check would fail?

Worked with @Enzo707 and resolved the issue

**Comment by QilongTang** [comment]
> ~I don't think there should there be any changes to package-lock json in this PR. Just the version update in the package.json.~. the build passes now

Cool, I was about to push my changes but I found you guys worked it out

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
@Enzo707 @avidit Please file a new Github release to trigger the release pipeline, there are plenty examples

**Pull request merged**